met-web/src/components/engagement/view/widgets/Subscribe/Subscribed/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ActionProvider } from '../../../ActionContext';
+import Subscription from './Subscription';
+
+export const ManageSubscription = () => {
+    return (
+        <ActionProvider>
```

Comment:
I don't think we should be using a unique ActionProvider more than once. If we need state in multiple pages, we should use Redux